src/org/parosproxy/paros/core/scanner/VariantJSONQuery.java

```diff
@@ -40,7 +40,11 @@
     public static final int BACKSLASH = '\\';
 
     private SimpleStringReader sr;
-
+
+    public VariantJSONQuery (int variantType) {
+    	super(variantType);

```

Comment:
Sorry, I was not clear, what I meant with the "use the new type always" was for the `VariantJSONQuery` have the type hardcoded (`super(NameValuePair.TYPE_JSON)`), not have the other parts of the code set the new type.